triplea_maps.yaml

```diff
@@ -2290,3 +2290,11 @@
   description: |
     <br>
     <br>This is Global 40 Expansion. Adds House Rules from The Captain and Company.<br>
+- mapName: ww2v4_variants
+  mapCategory: EXPERIMENTAL
+  url: https://github.com/triplea-maps/ww2v4_variants/archive/master.zip
+  version: 1
+  img: https://raw.githubusercontent.com/triplea-maps/ww2v4_variants/master/preview.png
```

Comment:
The image url is not quite right. It has to be with an uppercase P i.e. 'Preview.png' instead of 'preview.png', or instead you can change the filename in the repository itself

Reply:
Changing the file name is preferred, 'preview.png' will be the standard name and will be required in 2.6